app/controllers/search_controller.rb

```diff
@@ -0,0 +1,7 @@
+class SearchController < ApplicationController
+  def index
+    address = params[:location]
+    @station = NrelFacade.closest_station(address)
+    @route = MapquestFacade.route(address, @station.address)
```

Comment:
One refactor/design choice you could consider in the future is making one facade, which would call the MapquestService and the NrelService. That way, the facade would be responsible for aggregating data from both external APIs and encapsulating it into (probably) a single PORO.
I don't have a strong preference, but advantages would include 1) a simplified controller 2) fewer poros and facades to test.